d2common/d2data/d2datadict/monstats2.go

```diff
@@ -177,7 +177,7 @@ func LoadMonStats2(file []byte) {
 	d := d2common.LoadDataDictionary(file)
 	for d.Next() {
 		record := &MonStats2Record{
-			Key:             d.String("Id"),
+			Key:             d.String("ID"),
```

Comment:
we should not be altering the name of the column  we are retrieving from the txt files